client/components/breadcrumb/index.tsx

```diff
@@ -34,14 +35,23 @@ const StyledGridicon = styled( Gridicon )`
 
 interface Props {
 	items: { label: string; href?: string }[];
+	compact?: boolean;
 }
 
-const Breadcrumb: React.FunctionComponent< Props > = ( { items } ) => {
+const Breadcrumb: React.FunctionComponent< Props > = ( { items, compact = false } ) => {
+	const translate = useTranslate();
+	const back = translate( 'Back' ) as string;
+	if ( compact ) {
+		// Show only the exactly previous page
+		items = items.slice( -2, -1 );
+		items[ 0 ].label = back;
```

Comment:
Sorry for not commenting before, but there is any chance this `items[0]` to be `undefined`? We may have to do a check here.

Reply:
Good point, 
d9af329